source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -129,6 +130,13 @@ def _extend_journey(pt_journey, fallback_dp, fallback_extremity):
     pt_journey.distances.bike += aligned_fallback.journeys[0].distances.bike
     pt_journey.distances.car += aligned_fallback.journeys[0].distances.car
 
+    #Copy pt_section.origin to fallback_section.destination and pt_section.destination to fallback_section.origin
+    _, represents_start_fallback = fallback_extremity
+    if represents_start_fallback:
+        aligned_fallback.journeys[0].sections[0].origin.CopyFrom(pt_journey.sections[-1].destination)
+    else:
+        aligned_fallback.journeys[0].sections[0].destination.CopyFrom(pt_journey.sections[0].origin)
```

Comment:
There is probably something that I don't understand here, because this seems to be the inverse of what we should do.

Also the `aligned_fallback` can have multiple sections, typically when using bss.
And since it look like that [we insert the section at the then sort it later](https://github.com/kadhikari/navitia/blob/f255a334999fec696dc345f1897e5efc3d7dd32f/source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py#L202-L204), we have to be sure that the last section is the correct one.